Assets/Scripts/FengGameManagerMKII.cs

```diff
@@ -3464,8 +3475,7 @@ public void SpawnPlayerAt2(string id, GameObject pos)
             component.enabled = true;
             GameObject.Find("MainCamera").GetComponent<IN_GAME_MAIN_CAMERA>().setHUDposition();
             GameObject.Find("MainCamera").GetComponent<SpectatorMovement>().disable = true;
-            //TODO MouseLook
-            //GameObject.Find("MainCamera").GetComponent<MouseLook>().disable = true;
+            GameObject.Find("MainCamera").GetComponent<Assets.Scripts.UI.Camera.MouseLook>().disable = true;
```

Comment:
Import the namespace, and use MouseLook instead of Assets.Scripts.UI.Camera.MouseLook